devel/astyle/files/patch-Makefile.diff

```diff
@@ -1,58 +1,71 @@
---- Makefile.orig	2014-12-11 08:42:25.000000000 -0600
-+++ Makefile	2015-09-18 07:13:58.000000000 -0500
-@@ -33,14 +33,14 @@
+--- Makefile.orig	2018-10-21 15:13:22.000000000 -0400
++++ Makefile	2018-10-21 19:10:03.000000000 -0400
+@@ -36,23 +36,25 @@
  # define macros
  dylib = dylib
  dynamiclib = -dynamiclib
 +installname = -install_name ${prefix}/lib/
  bindir = bin
  objdir = obj
  ipath=$(prefix)/bin
- CBASEFLAGS = -W -Wall -fno-rtti -fno-exceptions
--UNIVFLAGS = -arch i386 -arch x86_64
- JAVAINCS   = -I/System/Library/Frameworks/JavaVM.framework/Headers
+ CBASEFLAGS = -W -Wall -std=c++11 -stdlib=libc++ -fno-rtti -fno-exceptions  \
+ -mmacosx-version-min=10.9
+-LDBASEFLAGS = 
++LDBASEFLAGS =
+ JAVAINCS  = -I/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/include \
+ -I/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/include/darwin \
```

Comment:
Why hardcode `jdk1.8.0_45.jdk`? I don't think most users would have this particular version installed.